esmvaltool/recipes/examples/recipe_check_obs.yml

```diff
@@ -244,6 +244,52 @@ diagnostics:
          type: reanaly, version: 1, start_year: 1948, end_year: 2018}
     scripts: null
 
+  OSI-450:
+    description: OSI 450 monthly data
+    variables:
+      sic:
+        preproc: nopp
+        mip: OImon
+        fx_files: [areacello]
```

Comment:
Note that this syntax no longer works since https://github.com/ESMValGroup/ESMValCore/pull/170 was merged. You can just add areacello as a normal variable.